src/config/database.ts

```diff
@@ -3,13 +3,15 @@ import dotenv from "dotenv";
 dotenv.config();
 
 export const sequelize = new Sequelize(
-    "eCommerceTap",
-    process.env.DB_USERNAME as string,
-    process.env.DB_PASSWORD,
-    {
-      host: process.env.DB_HOSTNAME,
-      dialect: process.env.DB_DIALECT as 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql',
-      port: Number(process.env.DB_PORT),
-    }
-  );
+  "ecommercetap",
+  process.env.DB_USERNAME as string,
+  process.env.DB_PASSWORD,
+  {
+    host: process.env.DB_HOSTNAME,
+    dialect:"mysql",
```

Comment:
very good, dialect should not be a variable

Reply:
This is the only thing I added to this, so thank you.